scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseRepository.java

```diff
@@ -136,8 +129,14 @@ public CouchbaseRepository(AsyncBucket bucket) {
                     new KeyNotFoundException(
                         String.format("Repository '%s' key '%s' not found", repository, key))))
         .map(AbstractDocument::content)
-        .flatMapIterable(JsonArray::toList)
-        .map(entry -> new HistoryDocument(currentVersion.incrementAndGet(), entry))
+        .flatMapIterable(
+            objects ->
+                objects.toList().stream()
+                    .map(
+                        value ->
+                            new HistoryDocument(
+                                currentVersion.incrementAndGet(), readJsonValue(value)))
```

Comment:
Why this way is better than the previous one?

Reply:
In previous on null value generated null point exception